lib/course/middleware/exceptions_middleware.rb

```diff
@@ -0,0 +1,18 @@
+module Course
+  module Middleware
+    class ExceptionsMiddleware
+      attr_reader :app
+
+      def initialize(app)
+        @app = app
+      end
+
+      def call(env)
+        status, headers, body = app.call(env)
+        [status, headers, body]
+      rescue StandardError => _e
+        [500, { "content-type" => "text/plain" }, ["unexpected error, try again"]]
```

Comment:
that would be strange, if you request a json and in return text/plain, or requested xml and have something meaningless to the requester